base/multimedia.jl

```diff
@@ -225,6 +225,18 @@ function displayable(m::MIME)
     return false
 end
 
+
+macro display(exs...)
+    blk = Expr(:block)
+    for ex in exs
+        push!(blk.args, :(print($(sprint(show_unquoted,ex)*" = "))))
+        push!(blk.args, :(display(begin value=$(esc(ex)) end)))
+        push!(blk.args, :(println()))
+    end
```

Comment:
Or you could call `show(STDOUT, "text/plain", x)` here, of course, but that would make the name `@display` something of a misnomer.